src/BuildIt.Forms/BuildIt.Forms.Controls/StatefulControl.xaml.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace BuildIt.Forms.Controls
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class StatefulControl
+    {
+        public static readonly BindableProperty StateProperty = BindableProperty.Create(nameof(State), typeof(StatefulControlStates), typeof(StatefulControl), propertyChanged: HandleStatePropertyChanged);
+
+        private const string EmptyStateContainer = nameof(EmptyStateContainer);
```

Comment:
To avoid name clashes should we change these constants to have "Name" as prefix (or another alternative) eg EmptyStateContainerName

Reply:
Yep, good idea